docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+* [REST API Compatibility] Nested path and filter sort options {es-pull}76022[#76022] (issues: {es-issue}42809[#42809], {es-issue}51816[#51816])
+* [REST API Compatibility] `CommonTermsQuery` and `cutoff_frequency` parameter {es-pull}75896[#75896] (issues: {es-issue}42654[#42654], {es-issue}51816[#51816])
+* [REST API Compatibility] Allow first empty line for `_msearch` {es-pull}75886[#75886] (issues: {es-issue}41011[#41011], {es-issue}51816[#51816])
+* Node level can match action {es-pull}78765[#78765]
+* TSDB: Add time series information to field caps {es-pull}78790[#78790] (issue: {es-issue}74660[#74660])
+* Add new kNN search endpoint {es-pull}79013[#79013] (issue: {es-issue}78473[#78473])
+* Disallow kNN searches on nested vector fields {es-pull}79403[#79403] (issue: {es-issue}78473[#78473])
+* Ensure kNN search respects authorization {es-pull}79693[#79693] (issue: {es-issue}78473[#78473])
+* Load kNN vectors format with mmapfs {es-pull}78724[#78724] (issue: {es-issue}78473[#78473])
+* Support cosine similarity in kNN search {es-pull}79500[#79500]
+* Node level can match action {es-pull}78765[#78765]
+* Check nested fields earlier in kNN search {es-pull}80516[#80516] (issue: {es-issue}78473[#78473])
+
+Security::
+* Add a tool for creating enrollment tokens {es-pull}74890[#74890]
+* Add the Enroll Kibana API {es-pull}72207[#72207]
+* Change default hashing algorithm for FIPS 140 {es-pull}55544[#55544]
+* Create enrollment token {es-pull}73573[#73573] (issues: {es-issue}71438[#71438], {es-issue}72129[#72129])
+* Enroll node API {es-pull}72129[#72129]
+* Not encoding the Api Key in Enrollment token {es-pull}74510[#74510] (issue: {es-issue}73573[#73573])
+* Configure security for the initial node CLI {es-pull}74868[#74868]
+* Adding base `RestHandler` class for Enrollment APIs {es-pull}76564[#76564] (issue: {es-issue}76097[#76097])
+* Generate and store password hash for elastic user {es-pull}76276[#76276] (issue: {es-issue}75310[#75310])
+* Set elastic password and generate enrollment token {es-pull}75816[#75816] (issue: {es-issue}75310[#75310])
+* Add `elasticsearch-enroll-node` tool {es-pull}77292[#77292]
+* Default hasher to `PBKDF2_STRETCH` on FIPS mode {es-pull}76274[#76274]
+* Add v7 `restCompat` for invalidating API key with the id field {es-pull}78664[#78664] (issue: {es-issue}66671[#66671])
+* Print enrollment token on startup {es-pull}78293[#78293]
+* Startup check for security implicit behavior change {es-pull}76879[#76879]
+* Update auto-generated credentials output {es-pull}79755[#79755] (issue: {es-issue}79312[#79312])
+* CLI tool to reconfigure nodes to enroll {es-pull}79690[#79690] (issue: {es-issue}7718[#7718])
+* Security auto-configuration for packaged installations {es-pull}75144[#75144] (issue: {es-issue}78306[#78306])
+* Update to OpenSAML 4 {es-pull}77012[#77012] (issue: {es-issue}71983[#71983])
+* URL option for `BaseRunAsSuperuserCommand` {es-pull}81025[#81025] (issue: {es-issue}80481[#80481])
```

Comment:
```suggestion
```

This was an enhancement over the alpha2 release

Reply:
Removed in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c.